examples/tools/use-cases/custom-strong-type-connection-package-tool-showcase/requirements.txt

```diff
@@ -0,0 +1,2 @@
+promptflow==0.1.0b8
+my-tools-package==0.0.3
```

Comment:
please use 0.0.5, since 0.0.3 has a typo in the MyCustomConnection comment, the api_key should be type Secret